WebApp/src/components/AddBlockForm/AddBlockForm.js

```diff
@@ -153,6 +158,10 @@ class AddBlockForm extends Component {
     onClose();
   }
 
+  handleChangeComplete = (color) => {
+    this.setState({blockColor: color.hex});
+  }
+
```

Comment:
Why are u using setState for blockColor two times in AddBlockForm ? It is on row 125 and 162 ?

Reply:
Because of text field which show code of color. User can change color of block in two places: from color palette but also from this text field.